listenbrainz/webserver/views/api_compat_deprecated.py

```diff
@@ -143,7 +147,7 @@ def _to_native_api(data, append_key):
                 'track_name': data['t{}'.format(append_key)],
                 'release_name': data['b{}'.format(append_key)],
                 'additional_info': {
-                    'source': data['o{}'.format(append_key)]
+                    'source': data.get('o{}'.format(append_key), '')
```

Comment:
This may add an empty key at times, which I would normally object to, but we don't actually store this data for a now listening request, right?

Reply:
We don't, but I'll remove it anyways.